src/NetMQ/NetMQRuntime.cs

```diff
@@ -38,7 +36,7 @@ public NetMQRuntime()
         /// </summary>
         public static NetMQRuntime Current
         {
-            get { return s_current.Value; }
+            get { return s_current.Value!; }
```

Comment:
This could return null if no instance of `NetMQRuntime` was created. I think we should check for null here and throw an exception that provides some more help than a NRE does.